rust/abacus-base/src/types/checkpoint_syncer.rs

```diff
@@ -54,10 +62,19 @@ pub struct MultisigCheckpointSyncerConf {
 
 impl MultisigCheckpointSyncerConf {
     /// Get a MultisigCheckpointSyncer from the config
-    pub fn try_into_multisig_checkpoint_syncer(&self) -> Result<MultisigCheckpointSyncer, Report> {
+    pub fn try_into_multisig_checkpoint_syncer(
+        &self,
+        origin: &str,
+        validator_checkpoint_index: IntGaugeVec,
+    ) -> Result<MultisigCheckpointSyncer, Report> {
         let mut checkpoint_syncers = HashMap::new();
         for (key, value) in self.checkpointsyncers.iter() {
-            checkpoint_syncers.insert(Address::from_str(key)?, value.try_into_checkpoint_syncer()?);
+            let gauge = validator_checkpoint_index
```

Comment:
I don't see this gauge getting initialized, but I could just be blind. We should be setting the current index as soon as possible ideally so that way if nothing changes for a while we are still reporting the metric after a reboot.

Reply:
I believe this will automatically get initialized since the `CheckpointFetcher` will fetch pretty much asap, and each syncers fetch will then set the gauge